### ISSUE TITLE
Previous pen gesture unresponsive compared to toggle eraser gesture - left handed mode

### ISSUE THREAD

**Issue by dfrl-dev** [ddvk/remarkable-hacks#504]
Version 2.14.3.1047-patch_37.2.01

Has anyone else experienced this?

It often takes me several swipes down in the top-right corner to switch to my previous pen type, but I have basically a 100% success rate on the gesture with the swipe up to toggle eraser.

**Comment by avillavecesn** [comment]
Yes, I have experienced exactly that. 100% success on the gesture to swipe up to toggle eraser, very limited success with the swipe down gesture to switch to my previous pen type...

**Comment by dfrl-dev** [comment]
Another note: 

The other two corner gestures (pen thickness, and previous colour) also work really well.  Of the four corner gestures only previous pen seems to have an issue.
